### PR TITLE
Visitor/User can View a Single Recipe

### PR DESCRIPTION
**Pull request by beccaburns**
Visitor/User can View a Single Recipe
===========================

[PT-Link](https://www.pivotaltracker.com/story/show/169667282)

## PR Contents
In this PR, a single recipe can be viewed.

## In this PR
- Links to navigate to single recipe
- Single recipe component 
- Axios request to backend for showing single recipe
- Unsuccessful message: "This recipe is currently unavailable.", status code: "404"
- Creates fixture for single recipe
- Integration spec file to test viewing single recipe
- Semantic UI updates

Screenshot 2019-11-13 at 12 22 13

### CODE REVIEW

**Review by miyeaier — APPROVED** [review]

File: cypress/fixtures/single_recipe.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{ 
+  "recipes":
```
Comment (tochman) [inline cypress/fixtures/single_recipe.json]: the response from the back-end is `recipe` not `recipes` when you return a single recipe.

File: cypress/fixtures/single_recipe.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  [
```
Comment (tochman) [inline cypress/fixtures/single_recipe.json]: It is also NOT an array. You need to check the request specs on the back end before creating fixtures.

File: cypress/fixtures/single_recipe.json (hunk 1 above)
Comment (clalil) [reply cypress/fixtures/single_recipe.json]: Our bad, resolved wit upcoming push.

File: cypress/fixtures/single_recipe.json (hunk 2 above)
Comment (clalil) [reply cypress/fixtures/single_recipe.json]: Our bad, lack of brain strength, resolved with upcoming push.

**Review by tochman — APPROVED** [review]
Looks good

**Review by yasmineezequiel — APPROVED** [review]

**Review by clalil — APPROVED** [review]

**Pull request merged**